### PR TITLE
New ActiveWorlds Cheat Sheet

### PR DESCRIPTION
**Pull request by AnthonyNeace**
IA: https://duck.co/ia/view/activeworlds_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@AnthonyNeace Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

While the team and the community review your submission, we'd like to make sure you look good! If you'd like a free DuckDuckGo tshirt, please send over the following info to merch@duckduckgo.com, and be sure to put "Dev Merch" in the subject line:

Name:
Street Address:
City:
State / Province:
Country:
Phone (Non-US Orders only): 
T-shirt size:
GitHub username:
Link to this Pull Request:

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/activeworlds.json [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+            {
+                "key": "[numpad *]",
+                "val": "camera zoom in"
+            },        
+            {
+                "key": "[numpad /]",
+                "val": "camera zoom out"
+            },      
+            {
+                "key": "[middle mouse]",
```
Comment (navneetzz) [inline share/goodie/cheat_sheets/json/activeworlds.json]: @AnthonyNeace do "[middle mouse]" to "middle mouse" . You don't need to add brackets if it's single value.

File: share/goodie/cheat_sheets/json/activeworlds.json [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                "val": "toggle mouselook mode on/off"
```
Comment (navneetzz) [inline share/goodie/cheat_sheets/json/activeworlds.json]: @AnthonyNeace add space between on / off

File: share/goodie/cheat_sheets/json/activeworlds.json [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+                "val": "rotate current selection 15 degrees on -y axis"
+            },  
+            {
+                "key": "home",
+                "val": "roll current selection 15 degrees on +z axis"
+            },  
+            {
+                "key": "end",
+                "val": "roll current selection 15 degrees on -z axis"
+            }
```
Comment (navneetzz) [inline share/goodie/cheat_sheets/json/activeworlds.json]: @AnthonyNeace capitalize first letter of first word in every value part.

File: share/goodie/cheat_sheets/json/activeworlds.json [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+                "val": "show tabs"
+            }         
+        ],     
+        "Object Properties": [
+            {
+                "key": "delete",
+                "val": "delete current selection"
+            },
+            {
+                "key": "[numpad 0]",
```
Comment (navneetzz) [inline share/goodie/cheat_sheets/json/activeworlds.json]: @AnthonyNeace here also remove brackets

File: share/goodie/cheat_sheets/json/activeworlds.json [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+                "val": "show whisper dialog"
+            }      
+        ],           
+        "World Interaction": [
+            {
+                "key": "[left mouse]",
+                "val": "activate object"
+            },
+            {
+                "key": "[right mouse]",
```
Comment (navneetzz) [inline share/goodie/cheat_sheets/json/activeworlds.json]: @AnthonyNeace remove brackets here also.

**Comment by moollaza** [comment]
@AnthonyNeace it looks like you've rebased this branch during/after making the PR? 

It now includes several other changes which should not be present. I would suggest you start over by closing this PR, resetting the branch and cherry picking the commits (or simply make a new commit with the current code).

How does that sound?

**Comment by AnthonyNeace** [comment]
@moollaza Sorry about that. New PR here: https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2200

**Pull request closed without merging**